main.js

```diff
@@ -1 +1,21 @@
+var submitForm = document.querySelector(".activity-button");
+var formView = document.querySelector(".left-section");
+var timerView = document.querySelector(".timer-container");
+var activityInput = document.querySelector(".text-field-long");
+var timeInput = document.querySelector(".time-inputs-sections")
+var categoryName = document.querySelector(".category-name");
 
+submitForm.addEventListener("click", submitActivity);
+
+function submitActivity(event) {
+  event.preventDefault();
+  formView.classList.add("hidden");
+  timerView.classList.remove("hidden");
```

Comment:
Opportunity for a dynamic function?

Reply:
replace 12-13 with :
function switchLeftDisplay{
   timerView.classList.toggle("hidden");
   formView.classList.toggle("hidden");
}

Reply:
Good idea!